internal/proto/data_coord.proto

```diff
@@ -350,6 +350,7 @@ message Binlog {
   uint64 timestamp_to = 3;
   string log_path = 4;
   int64 log_size = 5;
+  int64 logID = 6;
```

Comment:
log_id to keep format identical

Reply:
static checks will report logId is invalid after proto convert log_id to logId and recommend to use logID as a field name